dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImplTest.java

```diff
@@ -481,4 +487,157 @@ private void runLuceneQueryAndValidateResults(final String query, final Contentl
         assertEquals(contentlet.getInode(), hits[0].getSourceAsMap().get("inode"));
     }
 
+    @Test
+    public void test_findContentletByIdentifier() throws Exception {
+    
+        final Language language1 = new LanguageDataGen().nextPersisted();
+        final Language language2 = new LanguageDataGen().nextPersisted();
+        final ContentType blogType = TestDataUtils.getBlogLikeContentType(site);
+        
+
+        // create URL-Mapped content
+        final Contentlet workingOneLanguage = new ContentletDataGen(blogType.id())
+                .languageId(language1.getId())
+                .setProperty("body", "myBody")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "myBody" appears 4 times in this file; the first occurrence is on line 500](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5288908)